# Include paths shouldn't have consecutive double slashes.  Causes
# a problem for debugedit, used by rpmbuild to extract debuginfo.
diff -up meshlab-1.3.2/meshlab/src/meshlabplugins/render_splatting/splatrenderer.h.include-path-double-slash meshlab-1.3.2/meshlab/src/meshlabplugins/render_splatting/splatrenderer.h
--- meshlab-1.3.2/meshlab/src/meshlabplugins/render_splatting/splatrenderer.h.include-path-double-slash	2013-02-28 22:51:16.572824421 -0700
+++ meshlab-1.3.2/meshlab/src/meshlabplugins/render_splatting/splatrenderer.h	2013-02-28 22:51:54.298823186 -0700
@@ -25,7 +25,7 @@
 #define SPLATRENDERER_H
 
 #include <QObject>
-#include <common//interfaces.h>
+#include <common/interfaces.h>
 #include <wrap/gl/shaders.h>
 class QGLFramebufferObject;
 
diff -up meshlab-1.3.2/meshlab/src/meshlabplugins/render_splatting/splatrenderer_plugin.h.include-path-double-slash meshlab-1.3.2/meshlab/src/meshlabplugins/render_splatting/splatrenderer_plugin.h
--- meshlab-1.3.2/meshlab/src/meshlabplugins/render_splatting/splatrenderer_plugin.h.include-path-double-slash	2013-02-28 22:51:22.492824227 -0700
+++ meshlab-1.3.2/meshlab/src/meshlabplugins/render_splatting/splatrenderer_plugin.h	2013-02-28 22:52:05.766822811 -0700
@@ -25,7 +25,7 @@
 #define SPLATRENDERERPLUGIN_H
 
 #include <QObject>
-#include <common//interfaces.h>
+#include <common/interfaces.h>
 #include <wrap/gl/splatting_apss/splatrenderer.h>
 
 class QGLFramebufferObject;
